kube/ui-deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: ui
+    game: stacktrader
+  name: ui
+spec:
+  replicas: 1
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: ui
+        game: stacktrader
+    spec:
+      containers:
+        - env:
+            - name: PORT
+              value: "9090"
+            - name: RESGATE_HOST
+              value: http://resgate:8080
+          image: stacktrader/ui
+          name: ui
+          ports:
+            - containerPort: 9000
```

Comment:
The service is targeting port 9090 here, but you're using port 9000. Make sure that the nginx.conf file for the ui matches the port you're exposing here which matches the target port on the service. The `PORT` environment variable is not used by nginx.